designer/client/src/components/Menu/SubMenu.tsx

```diff
@@ -51,19 +51,33 @@ export function SubMenu({ slug }: Props) {
 
   function onFileUploaded(e: ProgressEvent<FileReader>) {
     const { result } = e.target ?? {}
+    const { current: input } = fileInput
 
-    // Reset input for next upload
-    if (fileInput.current) {
-      fileInput.current.value = ''
+    // Default filename for logging
+    let filename = 'file'
+
+    // Update filename and reset input for next upload
+    if (input?.files?.length) {
+      filename += ` '${input.files[0].name}'`
```

Comment:
Could this be simplified to `filename = input.files[0].name` rather than concatenating it with "file"?

Reply:
I've concatenated because `input.files` might not be available

### With file
>Upload file 'slug.json' failed

### Without file
>Upload file failed